assignment-4.rb

```diff
@@ -0,0 +1,80 @@
+class Vehicle
+
+    attr_accessor :wheels_count, :color, :price
+  
+    def initialize(wheels_count, color, price)
+      self.wheels_count = wheels_count
+      self.color = color
+      self.price = price
+    end
+  
+    def wheels
+      puts "This #{self.class} having #{wheels_count} wheels"
+    end
+  
+    def vehicle_color
+      puts "This #{self.class} having #{color} color"
+    end
+  
+    def vehicle_price
+      puts "This #{self.class} price is #{price}."
+    end
+  end
+  
+  class Car < Vehicle
+    def wheels
+      super
+      puts "This wheels are classic wheels"
+    end
+  
+    def vehicle_price
+      super
+      # state tax 10%
+      self.price+=self.price*0.1
+      # federal tax 5%
+      self.price+=self.price*0.05
+      # What is the model if it is a base model price is same as price entered
+      # select 1.Base 2.Mid 3.High
+      # if it is a mid range modal, we should add 10% more
+      # if it is a High end, we should add 25% more
+      puts "Select the model based upon the options"
+      puts "1.Base 2.Mid 3.High"
+      model=gets.chomp.to_i
+      case model
+      when 1
+
```

Comment:
What are you trying to do here when the condition is 1?

Reply:
As base model price is same as price entered, i left it without writing any condition. so that value of price will be unchanged.